src/app/components/login/login.component.spec.ts

```diff
@@ -20,4 +33,34 @@ describe('LoginComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should have a login as administrator anchor', () => {
+    const login: HTMLAnchorElement[] = fixture.nativeElement.querySelectorAll('.btn');
+    const loginAsAdmin = Array.from(login).find((el) => el.textContent === 'Login as Admin');
```

Comment:
Instead of `querySelectorAll` combined with finding by text, we should use a (unique) `data-testid` on both login buttons.